src/commands/Minion/checkpatch.ts

```diff
@@ -0,0 +1,77 @@
+import { CommandStore, KlasaMessage } from 'klasa';
+
+import { stringMatches, formatDuration, toTitleCase } from '../../lib/util';
+import { BotCommand } from '../../lib/BotCommand';
+import { Time, Emoji } from '../../lib/constants';
+import Farming from '../../lib/skilling/skills/farming/farming';
+import { requiresMinion } from '../../lib/minions/decorators';
+import resolvePatchTypeSetting from '../../lib/farming/functions/resolvePatchTypeSettings';
+import { PatchTypes } from '../../lib/farming';
+import { FarmingPatchTypes } from '../../lib/farming/types';
+
+export default class extends BotCommand {
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			altProtection: true,
+			oneAtTime: true,
+			cooldown: 1,
+			usage: '',
+			usageDelim: ' '
+		});
+	}
+
+	@requiresMinion
+	async run(msg: KlasaMessage) {
+		await msg.author.settings.sync(true);
+		const currentDate = new Date().getTime();
+
+		let baseStr = '';
+		let emojiStr = '';
+		let contentStr = '';
+		let finalStr = '';
+
+		for (const patches in FarmingPatchTypes) {
```

Comment:
`for in` loops are not preferrable and you should always use `for of` loop first, and then a regular `for` loop if that doesnt work.